Multiple tasks about multi-catch
<p>----------------------------------------</p>
How multi-catch works? If an exception occurs in a try block, execution of the program jumps to the first catch. If the type specified inside the catch statement matches the exception type, then the code inside the {} block starts executing. Otherwise, execution moves to the next catch and the check is repeated. Now let's practice to help reinforce this concept.
<p>----------------------------------------</p>
<p>"Hi, Amigo. Yesterday I was a bit tipsy, so I'm afraid I've messed up your tasks. But I hope you wonâ€™t be mad at me and will complete them all. It's for your own good. Here they come."</p>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0913"></div>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0914"></div>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0915"></div>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0916"></div>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0917"></div>
<div class="task-widget-container" showCover="true" taskKey="com.codegym.task.task09.task0918"></div>